src/Components/Browser.JS/dist/Debug/blazor.webassembly.js

```diff
@@ -316,19 +316,15 @@ var DotNet;
             promise.catch(function (error) { return console.error(error); });
         };
         DotNetObject.prototype.serializeAsArg = function () {
-            return "__dotNetObject:" + this._id;
+            return { __dotNetObject: this._id };
```

Comment:
Is this file supposed to be checked in?

Reply:
Yes. We landed on checking this file in and adding a code-check to make sure it's up to date with the ts files rather than produce it as part of every build. Some of our build environments don't have `yarn` and there were some build from source concerns where we wanted to avoid `yarn`